src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/DlsFlsValveImpl.java

```diff
@@ -253,4 +260,58 @@ public void accept(StringTerms.Bucket bucket) {
             this.bucket = bucket;
         }
     }
+
+    private static class StringTermsGetter {
+        private static final Field REDUCE_ORDER = getField(InternalTerms.class, "reduceOrder");
+        private static final Field TERM_BYTES = getField(StringTerms.Bucket.class, "termBytes");
+        private static final Field FORMAT = getField(InternalTerms.Bucket.class, "format");
+
+        private StringTermsGetter() {
+        }
```

Comment:
nit: single line {}?

Reply:
I don't think it is a common style to have single line `{}`.